python35_filter.cpp

```diff
@@ -25,6 +27,224 @@
 
 using namespace std;
 
+/**
+ * Python filter initialisation
+ */
+void Python35Filter::init()
+{
+	// Embedded Python 3.5 program name
+	wchar_t *programName = Py_DecodeLocale(m_name.c_str(), NULL);
+	Py_SetProgramName(programName);
+	PyMem_RawFree(programName);
+
+	// Embedded Python 3.5 initialisation
+	PythonRuntime::getPythonRuntime();
+
+	m_init = true;
+
+	PyGILState_STATE state = PyGILState_Ensure(); // acquire GIL
+
+	// Pass Fledge Data dir
+	setFiltersPath(getDataDir());
+
+	// Set Python path for embedded Python 3.5
+	// Get current sys.path. borrowed reference
+	PyObject* sysPath = PySys_GetObject((char *)string("path").c_str());
```

Comment:
Do we need to create a string object, can't we just use ' (char *)"path" ' ?

Reply:
I suspect we don't this is old code that can probably be removed but I would rather do that sort of thing in a future change where we are not in as much of a rush to satisfy a customer.